text/bench_test.go

```diff
@@ -19,6 +19,7 @@ import (
 	"io"
 	"io/ioutil"
 	"testing"
+
 	dto "github.com/prometheus/client_model/go"
```

Comment:
To fix this properly (according to the Prometheus style guide) this has to go to the bottom most group.

(The original placing was a glitch, and older gofmt liked it there.)